pages/draft/2020-12/release-notes.md

```diff
@@ -155,60 +155,64 @@ Here's how you would covert a schema using `$recursiveRef` to use `$dynamicRef`.
   </tr>
   <tr>
     <td>
-      ```json
-        // tree schema, extensible
-        {
-          "$schema": "https://json-schema.org/draft/2019-09/schema",
-          "$id": "https://example.com/tree",
-          "$recursiveAnchor": true,
-          "type": "object",
-          "properties": {
-            "data": true,
-            "children": {
-              "type": "array",
-              "items": { "$recursiveRef": "#" }
-            }
-          }
-        }
-        // strict-tree schema, guards against misspelled properties
-        {
-          "$schema": "https://json-schema.org/draft/2019-09/schema",
-          "$id": "https://example.com/strict-tree",
-          "$recursiveAnchor": true,
-          "$ref": "tree",
-          "unevaluatedProperties": false
-        }
-      ```
-    </td>
+
```

Comment:
Why doing changes with this file ?

Reply:
These changes are for another issue.

Reply:
how can i remove the changes of this file and keep the ones for this issue only in this pull request